ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -22438,16 +22414,6 @@ periodics:
     job-release: "4.12"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-azure-ipi-fips-p3-f1-openshift-logging-5-6
-  reporter_config:
```

Comment:
@jianzhangbjz is it correct to remove it?
as I known, it is added by writing this file directly, and not added by `make update`. for example https://github.com/openshift/release/pull/35998/files
so, maybe `make update` remove them.
you could check if they are needed. if yes, do not remove them and maybe manually add them back after `make update`.

here removed:
channel: '#forum-prow-hive-qe-ci'
channel: '#aos-qe-logging'

Reply:
Yes, they were removed by the `make update`. I'm not sure why they add them in the `job`, they should add in the `config`, otherwise, it will be removed when running `make update`.

Reply:
Report a bug to DPTP team: https://issues.redhat.com/browse/DPTP-3366